recipes/clinker/clinker-wrapper.sh

```diff
@@ -15,8 +15,6 @@ END
 )
 # Clinker wrapper - invoke bpipe
 
-#bpipe -p option1="something" -p option2="something_else" [...] $CLINKERDIR/workflow/clinker.pipe /path/to/*.fastq.gz
-# parse and accept parameters in bpipe style, then pass on to fixed clinker.pipe location
```

Comment:
It appears this entire file should be distributed upstream. Can you create a PR there and link it in the recipe?

Reply:
This recipe has been revisited in #25647 . Though will note here, this is same approach as for the [JAFFA](https://github.com/bioconda/bioconda-recipes/tree/master/recipes/jaffa) recipe with wrapper script seperate from source.